src/main/docs/guide/ioc/android.adoc

```diff
@@ -1,3 +1,5 @@
+// To move this to multi-lang, we would need to import android deps in build.gradle, and I'm unsure how we'd test it
+// It would be cool to have multi-lang capabilities that don't compile for situations like this
```

Comment:
We can come back to this after the rest of the docs are done. You can remove the comment